bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx88-en.html

```diff
@@ -0,0 +1,143 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_88_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      <h2 class="c-page-header-logo-fx">{{ ftl('whatsnew-firefox-browser') }}</h2>
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>Congrats! You’re using now version <span data-audio="/media/img/firefox/whatsnew/whatsnew88-en/jingle.mp3" class="flux" id="outatime">88</span> of Firefox.</p>
```

Comment:
I'm assuming this is only a small mp3 file, but do we need to consider where to host this seeing as it's on a /whatsnew page and potentially high-traffic?

Reply:
@pmac Should I get this out of this repo and into the assets.moz.net?

Reply:
I think it's fine. It's 26k, so it's smaller than most of the images on the page and should be downloaded far less often.